WordPress/WordPressApi/WordPressComRestApi.swift

```diff
@@ -27,7 +27,7 @@ public class WordPressComRestApi: NSObject
 
     public static let apiBaseURLString: String = "https://public-api.wordpress.com/rest/"
 
-    private let oAuthToken: String?
+    public let oAuthToken: String?
```

Comment:
The problem is that if your see other PR that I have on the review queue, we have several place where we need to get the authtoken to use it on non-related API calls, like downloading of image on private sites or start the Stats services, that have a separate API.